archunit/src/test/java/com/tngtech/archunit/library/plantuml/rules/JavaClassDiagramAssociationTest.java

```diff
@@ -82,22 +68,6 @@ public void reports_if_class_is_contained_in_any_component() {
                 .as("association contains " + File.class.getName()).isFalse();
     }
 
-    @Test
-    public void class_resides_in_multiple_packages() {
-        JavaClassDiagramAssociation javaClassDiagramAssociation = createAssociation(TestDiagram.in(temporaryFolder)
-                .component("A").withStereoTypes("..foopackage..")
-                .component("B").withStereoTypes("..barpackage")
-                .write());
-        JavaClass classContainedInTwoComponents = importClassWithContext(ClassInFooAndBarPackage.class);
-
-        thrown.expect(ComponentIntersectionException.class);
```

Comment:
We could keep the test case, but document the new behavior that a `IllegalArgumentException` is thrown.